draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/balistic.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.5
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{"Sir! They're here again!"}
+
+\textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}
+
+\textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}
```

Comment:
```suggestion
\textit{The time has come to end these long-standing wars and crush your enemies once and for all!}
```

IMO, "like a bug" doesn't fit with Heroes setting.

Reply:
Actually i added it cuz in HOMAM IV was that like "Ajwar will crush enemy like a bug." Little easter egg :)

Reply:
Oh, please add a comment with this. It should be a note to translators.